activerecord/test/models/company.rb

```diff
@@ -26,6 +26,9 @@ def arbitrary_method
   def private_method
     "I am Jack's innermost fears and aspirations"
   end
+
+  class SpecialCo < Company
+  end
```

Comment:
When I attempt to replace my model with that, I receive this error:

```
InheritanceTest#test_new_without_storing_full_sti_class:
ActiveRecord::SubclassNotFound: Invalid single-table inheritance type: Firm is not a subclass of Namespaced::Company
```

I'm not sure exactly what causes that failure. Any thoughts?